.github/workflows/goreleaser.yaml

```diff
@@ -0,0 +1,96 @@
+name: goreleaser
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+  pull_request: {}
+defaults:
+  run:
+    shell: bash
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # GoReleaser requires fetch-depth: 0 for
+          # changelog generation to work correctly.
+          fetch-depth: 0
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '~1.16'
+
+#      - name: "Download osxcross cross-compiler for macOS builds"
+#        run: |
+#          git clone https://github.com/tpoechtrager/osxcross /tmp/osxcross
+#          cd /tmp/osxcross
+#          wget -P tarballs '<REPLACE_ME_WITH_SECRET_PATH_TO_MACOS_SDK>'
+#          echo "/tmp/osxcross/target/bin" >> $GITHUB_PATH
+#
+#      - name: "Cache the osxcross + tarball build output"
+#        id: cache-osxcross
+#        uses: actions/cache@v2
+#        with:
+#          path: /tmp/osxcross/target
+#          key: ${{ runner.os }}-osxcross-${{ hashFiles('/tmp/osxcross/**/*') }}
+#          restore-keys: |
+#            ${{ runner.os }}-osxcross-
+#
+#      - name: "Build osxcross"
+#        if: steps.cache-osxcross.outputs.cache-hit != 'true'
+#        run: |
+#          cd /tmp/osxcross
+#          sudo ./tools/get_dependencies.sh
+#          UNATTENDED=1 ./build.sh
```

Comment:
for cleanliness, could we drop disabled config with an additional commit?

(dogfooding some feedback I've dished out to other contributors in the past)

Reply:
I'd also prefer this approach :+1:

Reply:
We can, but we'll have to merge this PR to keep the commits intact.

/label tide/merge-method-merge

Reply:
Done.